scripts/zap-scanner.sh

```diff
@@ -58,6 +58,12 @@ ZAP_CLI_OPTIONS="\
   -config globalexcludeurl.url_list.url\(0\).regex='.*/robots\.txt.*' \
   -config globalexcludeurl.url_list.url\(0\).description='Exclude robots.txt' \
   -config globalexcludeurl.url_list.url\(0\).enabled=true \
+  -config globalexcludeurl.url_list.url\(1\).regex='^https?://.*\.cdn\.mozilla\.(?:com|org|net)/.*$' \
```

Comment:
added 2 sets of URLs to global exclusions:
- URL that ends in `cdn.mozilla.net` (non-TDP site [ref](https://github.com/zaproxy/zaproxy/issues/6443#issuecomment-775226449))
- URL that ends in `amazonaws.com` (S3 buckets)

Reply:
So this is effective? We were able to resolve the issues?

Reply:
yep ([example](https://17074-281707402-gh.circle-artifacts.com/0/tdrs-backend/reports/owasp_report.html)). still increased the timeout since this step in the workflow still takes a while.